src/Application.php

```diff
@@ -2,99 +2,140 @@
 
 namespace DockerHostManager;
 
+use Docker\Container;
 use Docker\Docker;
+use Docker\Exception\ContainerNotFoundException;
 use Docker\Http\DockerClient;
+use Docker\Manager\ContainerManager;
 
 class Application
 {
+    const START_TAG = '## docker-hostmanager-start';
+    const END_TAG = '## docker-hostmanager-end';
+
+    /** @var string */
     private $entrypoint;
+    /** @var string */
     private $hostsFile;
-    private $tld;
 
+    /** @var DockerClient */
     private $client;
-    private $docker;
+    /** @var ContainerManager  */
+    private $containerManager;
+
+    /** @var array Container */
+    private $activeContainers = [];
 
+    /**
+     * @param string $entrypoint
+     * @param string $hostsFile
+     */
     public function __construct($entrypoint, $hostsFile, $tld)
     {
         $this->entrypoint = $entrypoint;
         $this->hostsFile = $hostsFile;
-        $this->tld = $tld;
```

Comment:
It wasn't used in my code anymore. But I have just seen that I forgot to append it to the container name in /etc/hosts.